### ISSUE TITLE
Sugestions to improve project initizalization

### ISSUE THREAD

**Issue by naaando** [vanat/vanat#4]
TL;DR
* Use current system username and git config (~/.gitconfig) to make sugestions
* Present numbered options instead of typing it
* Join username with package name
* Use stdin.read () instead of stind.read_line () to fix #3 

### Symbols
[...]  ->  default input in case of enter
 ...   ->  arbitrary input by user

### Init command

This command will guide you through creating your vanat.json config. 

`use current system username and git config (~/.gitconfig) to make sugestions`
\* First time configuring user
username (github username, nickname...) [derp]: ...
full name ["Erp derp"]: ...
email ["erpderp@foo.corporation"]: ...

\* Configuring package
Package name: ... `Join username with package name`
Description: ... `Use stdin.read () instead of stind.read_line () to fix #3 `
Version: ...

`Present numbered options instead of typing it`
Package Type:
[1] Project [2] Library [3] Plugin
...
License: 
[1] GPL3 [2] MIT [3] Proprietary [4] Other
...

**Comment by robertsanseries** [comment]
- [ ] Use current system username and git config (~/.gitconfig) to make sugestions
- [ ] Present numbered options instead of typing it
- [x] Join username with package name
<strike>Use stdin.read () instead of stind.read_line ()</strike> **Does not work. this same problem is presented in composer**
